daml-assistant/src/DAML/Assistant/Types.hs

```diff
@@ -76,6 +76,7 @@ newtype UserCommandArgs = UserCommandArgs
 -- | Command-line options for daml version command.
 data VersionOptions = VersionOptions
     { vAll :: Bool -- ^ list all available versions
```

Comment:
Both these options to me feel like `--verbose`. Do we need separate options for them?

Reply:
I like the idea of having individual toggles for showing different things. Adding a `--verbose` option that turns everything on by default isn't a bad idea though.